Pipfile

```diff
@@ -22,10 +22,11 @@ flask-cors = "*"
 click-log = "*"
 joblib = "*"
```

Comment:
I think we should pin `joblib` too, since it's used to serialize some data to disk. It would be unfortunate if the version suddenly changed when an upstream version is released, and the serialized files couldn't be loaded anymore.

Reply:
Pinned `joblib==0.13.2`.